src/libraries/Microsoft.PowerFx.Connectors/Execution/FormulaValueSerializer.cs

```diff
@@ -123,39 +123,46 @@ private void WriteProperty(string propertyName, OpenApiSchema propertySchema, Fo
             switch (propertySchema.Type)
             {
                 case "array":
-                    // array
-                    if (fv is not TableValue tableValue)
+
+                    if (fv is BlankValue)
                     {
-                        throw new ArgumentException($"Expected TableValue and got {fv?.GetType()?.Name ?? "<null>"} value, for property {propertyName}");
+                        StartArray(propertyName);
+                        WriteValue(null);

```

Comment:
>WriteValue(null); [](http://example.com/codeflow?start=24&length=17)

Should this be WriteStringValue(string.Empty) ?
Blank Array in json is `[]`, while this gives `[null]` = [Blank()]

Reply:
Blank will print no property at all.